### ISSUE TITLE
Faulty option parsing in line-comment-position

### ISSUE THREAD

**Issue by meeber** [eslint/eslint#7100]
**Tell us about your environment**

* **ESLint Version:** 3.5.0
* **Node Version:** 6.5.0
* **npm Version:** 3.10.3

**What parser (default, Babel-ESLint, etc.) are you using?**

Default

**Please show your full configuration:**

```yaml
rules:
  line-comment-position:
    - error
    - above
```

**What did you do? Please include the actual source code causing the issue.**

Linted the following file:

```js
// Meep
console.log("Meep");
```

**What did you expect to happen?**

For there not to be any linting errors.

**What actually happened? Please include the actual, raw output from ESLint.**

There was a linting error:

```
1:1  error  Expected comment to be beside code  line-comment-position
```

The cause of the problem is a typo [here](https://github.com/eslint/eslint/blob/6cfe5195caaa16c4e310d51ba54c6523444a47df/lib/rules/line-comment-position.js#L53). `typeof option` should be `typeof options`.

**Comment by platinumazure** [comment]
Confirmed in online demo. Thanks for the report, I'll work on a fix!
